kabuki/analyze.py

```diff
@@ -476,9 +476,21 @@ def _plot_posterior_pdf_node(bottom_node, axis, value_range=None, samples=10, bi
 
     # Plot data
     if len(bottom_node.value) != 0:
-        axis.hist(bottom_node.value.values, density=True, color='r',
-                  range=(value_range[0], value_range[-1]), label='data',
-                  bins=bins, histtype='step', lw=2.)
+        data_processor = kwargs.pop('data_preprocessor', None)
```

Comment:
Need to add this kwarg with an explanation to the doc string.

Reply:
added to doc for `plot_posterior_predictive()`.